config/coreboot-qemu.config

```diff
@@ -1,18 +1,19 @@
 CONFIG_ANY_TOOLCHAIN=y
 # CONFIG_INCLUDE_CONFIG_FILE is not set
-CONFIG_CBFS_SIZE=0x700000
+CONFIG_CBFS_SIZE=0xF00000
 # CONFIG_POST_IO is not set
 CONFIG_BOARD_EMULATION_QEMU_X86_Q35=y
 # CONFIG_POST_DEVICE is not set
+CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="qemu-coreboot"
```

Comment:
Should not be present. Injected by https://github.com/osresearch/heads/blob/bd7a945bbbf586ceb7c83785ef36d95f0adf7a15/modules/coreboot#L41